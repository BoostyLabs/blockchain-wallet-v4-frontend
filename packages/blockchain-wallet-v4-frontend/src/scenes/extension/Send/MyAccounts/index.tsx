import React from 'react'

import { WalletsWrapper } from 'blockchain-wallet-v4-frontend/src/scenes/extension/Send'
import WalletItem from 'blockchain-wallet-v4-frontend/src/scenes/extension/Send/WalletItem'

const MyAccounts: React.FC = () => {
  // TODO: mock accounts wallets data.
  const myAccounts: string[] = [
    'A4Pd7dB08D75679bbE90b7B1eDfB8BE3a16897Ee77',
    'B4Ad7dB08D75679bbE90b7B1eDfB8BE3a16897Ee77'
  ]

  return (
    <WalletsWrapper>
      {myAccounts.map((address: string) => (
        <WalletItem walletAddress={address} key={address} />
      ))}
    </WalletsWrapper>
  )
}

export default MyAccounts
