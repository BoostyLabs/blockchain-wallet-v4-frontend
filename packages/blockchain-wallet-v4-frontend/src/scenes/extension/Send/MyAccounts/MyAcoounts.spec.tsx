import React from 'react'
import MyAccounts from 'blockchain-wallet-v4-frontend/src/scenes/extension/Send/MyAccounts'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('My accounts extension tab', (): void => {
  const container = shallow(<MyAccounts />)
  it('My accounts tab renders correctly', (): void => {
    const tree = toJson(container)
    expect(tree).toMatchSnapshot()
  })
})
