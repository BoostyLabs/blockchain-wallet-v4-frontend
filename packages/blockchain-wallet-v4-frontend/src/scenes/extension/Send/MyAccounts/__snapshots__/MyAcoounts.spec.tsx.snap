// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My accounts extension tab My accounts tab renders correctly 1`] = `
<styled.div>
  <WalletItem
    key="A4Pd7dB08D75679bbE90b7B1eDfB8BE3a16897Ee77"
    walletAddress="A4Pd7dB08D75679bbE90b7B1eDfB8BE3a16897Ee77"
  />
  <WalletItem
    key="B4Ad7dB08D75679bbE90b7B1eDfB8BE3a16897Ee77"
    walletAddress="B4Ad7dB08D75679bbE90b7B1eDfB8BE3a16897Ee77"
  />
</styled.div>
`;
